providers/universal-provider/test/index.spec.ts

```diff
@@ -1001,6 +1002,78 @@ describe("UniversalProvider", function () {
           expectedChainId,
         });
       });
+      it("should cache `wallet_getCapabilities` request", async () => {
```

Comment:
do you need a negative test here if not cached?

Reply:
the first part of the test where the wallet receives the request covers the not cached part. If it was cached already the request wouldn't have been sent to the wallet